common/config.ts

```diff
@@ -112,3 +112,7 @@ export function findValueName(rows: ValueName[], value: string) {
     if (row == undefined) return "N/A"
     return row.name
 }
+
+export const EXTRA_MISSION_HOUSES: string[] = ["Riches", "Samarama"]
```

Comment:
It would be better to add some boolean flags to the votingRight and not match according to name.

Reply:
Hmm ok so when we assign the house to the delegate, then add flags. I will check it out.

Reply:
But even when we add the flags at the assignment of who controls the neutral house, we would need to match based on the name.

Reply:
Fixed: extraMission or extraUnit is added for voting right in the landsraad ui, and saved as DB variable for votingRight and then read from there when generating actions.